### PR TITLE
Change Gnosis safe wallet notice text add link

### PR DESCRIPTION
**Pull request by samuveth**
Based off https://github.com/snapshot-labs/snapshot/pull/891

Changes:
- Rename to ModalNotice and made modular to be used for other notices
- Change Gnosis wallet notice text and provide link to delegate page

### CODE REVIEW

**Comment by samuveth** [comment]
> Looks good to me

Then please approve ;)

**Comment by samuveth** [comment]
Based of https://github.com/snapshot-labs/snapshot/pull/891
Please merge that first.

File: src/locales/default.json [hunk 1]
```diff
@@ -63,8 +63,8 @@
   "seeInExplorer": "See on explorer",
   "learnMore": "Learn more",
   "logout": "Log out",
-  "walletNotice": "Wallet not supported",
-  "gnosisSafeWalletNotice": "You are trying to use Gnosis Safe wallet, which doesn't support signing messages on Snapshot.",
+  "walletNotice": "Wallet notice",
+  "gnosisSafeWalletNotice": "Gnosis Safe wallet doesn't support signing messages, you can delegate your voting power on our delegation page.",
```
Comment (midgerate) [inline src/locales/default.json:67]: Gnosis Safe wallet doesn't support signing messages. You can delegate your voting power to a wallet of your choice on our delegation page.

**Comment by midgerate** [comment]
@samuveth good from my side.

**Review by mktcode — APPROVED** [review]

**Pull request merged**